test/distributed/org/apache/cassandra/distributed/impl/AbstractCluster.java

```diff
@@ -191,6 +193,13 @@ public B withNodeProvisionStrategy(INodeProvisionStrategy.Strategy nodeProvision
             this.nodeProvisionStrategy = nodeProvisionStrategy;
             return (B) this;
         }
+
+        @Override
+        public C createWithoutStarting() throws IOException
+        {
+            Assume.assumeFalse("vnode is not supported", !isAllowVnodes() && getTokenCount() > 1);
```

Comment:
in the dtest api we have

```
if (tokenSupplier == null)
             tokenSupplier = evenlyDistributedTokens(nodeCount);
             tokenSupplier = evenlyDistributedTokens(nodeCount, tokenCount);
```

so we only care about token count IFF token supplier isn't provided... so only thing I can think of is to also include

`getTokenSupplier().tokens(1).size() == 1` then not vnodes